javascript-data-model/main.js

```diff
@@ -0,0 +1,104 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    {
+      orderNumber = 11439416898772232,
+      orderPlaced = '8/4/2020',
+      total = 34.00,
+      shipTo = 'JS Masher'
+    },
+    [
```

Comment:
you don't want to have another array here. you only need one array to contain all of the objects.